### PR TITLE
 #167164991 like and dislike api endpoint

### PR DESCRIPTION
**Pull request by WilliamsOhworuka**
#### Pivotal tracker story
[#167164991](https://www.pivotaltracker.com/story/show/167164991)

#### What does this PR do?
   - Creates an endpoint for liking and unliking Novels
 
#### Summary of Task
   - Design database structure and a holistic approach to implementing the feature
   - Think about possible edge cases and write tests for each instance
   - Come up with the algorithm for liking and unliking Novels
   - Create and add the necessary middlewares, helpers, and controllers
   - Create `like and unlike novel` endpoint

#### How can this be tested?
   - pull the branch to your local repository: `git pull origin/feature/167164991/like-dislike-api-endpoint`
   - install packages and dependencies: `npm install`
   - run automated tests: `npm run mocha-test`
   - add authorization header (signin on postman using `api/v1/users/login` copy the token from the response body and add it to the authorization field of the route below)
   - test manually on Postman: `/api/v1/users/:novelId`

### CODE REVIEW

File: src/database/models/novels.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default (sequelize, DataTypes) => {
```
Comment (Nerocodes) [inline src/database/models/novels.js]: There's already a novel model and migration with the required columns and association, you should use that and not create a new one

**Comment by Ukhu** [comment]
Kindly rename your PR in this format: `#[STORY_ID] Story description`

File: src/database/models/novels.js (hunk 1 above)
Comment (WilliamsOhworuka) [reply src/database/models/novels.js]: okay
